pyproject.toml

```diff
@@ -31,3 +31,24 @@ exclude_lines = [
 
 [tool.coverage.html]
 directory = "parts/htmlcov"
+
+[tool.mypy]
+files = "src/roman"
+strict = true
+strict_bytes = true
+local_partial_types = true
+python_version = "3.9"
```

Comment:
At least this line should be generated to be updated automatically when dropping Python 3.9 support sometime in 2026.